manim/utils/tex_file_writing.py

```diff
@@ -177,11 +195,11 @@ def compile_tex(tex_file, tex_compiler, output_format):
                 if error_pos:
                     with open(tex_file, "r") as g:
                         tex = g.readlines()
-                        logger.error("LaTeX compilation error! LaTeX reports:")
+                        logger.error("LaTeX compilation error: {log[error_pos][2:])}")
                         for log_index in error_pos:
                             index_line = log_index
-                            CONTENT = (
-                                f"{log[log_index][2:]}Here is the tex content:\n\n"
+                            context = (
+                                f"Context for error:\n\n"
```

Comment:
```suggestion
                                "Context for error:\n\n"
```

To make `flake8` pass